inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -130,11 +132,41 @@ void visit_exec_graph_node(pugi::xml_node& data, std::string& node_type_name,
     }
 }
 
+std::vector<std::shared_ptr<Node>> get_ordered_ops(const ngraph::Function& f) {
```

Comment:
Can we change the logic inside the `function->get_ordered_ops()` method?
Or do we have some logic which depends on it?

Reply:
We have a talk about it. At the current moment we don't see any dependence on ops order. I'll add new commit with order change to check if all CI tests will pass. 

@tomdol @postrational what do you think? Can we change this order without breaking anything?

Reply:
We can change the default way of sorting ops in ngraph::Function but it definitely has to be validated really well before we do it. A full set of CI checks is the minimum that should be tested with such a change.

There's another option though - the ngraph::Function lets you specify a custom sorting algorithm https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/ngraph/function.hpp#L132 - you can just set your own sorter on a function that you're processing in this serialization pass.

Reply:
I strongly suggest to avoid changes in `f->get_ordered_ops()` I know cases when people relies on topological sort order.
So it must be another version of this `get_ordered_ops` method. But anyway lets keep this implementation only for Serialization purposes because it is very MO specific sort.

Reply:
Ok so I won't be changing f->get_ordered_ops() and will leave get_ordered_ops() as a private serialization function.